DesktopUI2/DesktopUI2/ViewModels/StreamViewModel.cs

```diff
@@ -1014,7 +1014,7 @@ public async Task DownloadImage(string url)
       }
       catch (Exception ex)
       {
-        Serilog.Log.Debug(ex, "Swallowing exception {exceptionMessage}", ex.Message);
+        Serilog.Log.Warning(ex, "Swallowing exception {exceptionMessage}", ex.Message);
```

Comment:
There is a risk this would introduce a lot of noise being an HTTP method, no?

Reply:
Possibly yes, but I'd add this for now. Its just warning, so its not polluting the Sentry event log, but we can have a better idea, what is going wrong.

Also info and debug messages are retained for a short period of time on the `seq` server. It's a bit better to keep these for a while, while we act on the results.